frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.html

```diff
@@ -8,8 +8,11 @@
      [attr.aria-label]="text.query_has_changed_click_to_save"
      [attr.title]="text.query_has_changed_click_to_save">
     <span aria-hidden="true" class="icon icon-save"></span>
-  </a>
+  </a> 
+
   <input #editableTitleInput
+  id="inputTextID"
```

Comment:
This ID is not very specific. An ID should short, but precisely describe the field. `inputTextID` could be any input field within the application. However, I think there is no ID needed at all, please see my comment below regarding the styles.